components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/behavior/BrowserToolbarBottomBehavior.kt

```diff
@@ -46,6 +48,18 @@ class BrowserToolbarBottomBehavior(
         duration = SNAP_ANIMATION_DURATION
     }
 
+    /**
+     * Reference to [EngineView] used to check user's [android.view.MotionEvent]s.
+     */
+    private var engineView: EngineView? = null
+
+    /**
+     * Only scroll for certain [EngineView.InputResult] responses.
+     * See https://github.com/mozilla-mobile/fenix/issues/8768#issuecomment-592718468
```

Comment:
nit: We don't link to external projects within our source. Maybe we could paraphrase that comment here?

Reply:
I propose:

> Depending on how user's touch was consumed by GeckoView / current website,
we will animate the dynamic navigation bar if:
> - touches were used for zooming / panning operations in the website.
>
> We will do nothing if:
> - the website is not scrollable
> - the website handles the touch events itself through it's own touch event listeners.

Reply:
Looks great!

minor nit: `GeckoView` -> `EngineView`